Conversions/HexaDecimalToDecimal.java

```diff
@@ -0,0 +1,43 @@
+package Conversions;
+
+import java.util.Scanner;
+
+public class HexaDecimalToDecimal {
+
+    // convert hexadecimal to decimal
+    public static int getHexaToDec(String hex){
+        String digits = "012345678910ABCDEFF";
+        hex = hex.toUpperCase();
+        int val = 0;
+        for (int i = 0; i < hex.length(); i++)
+        {
+            char c = hex.charAt(i);
```

Comment:
* This line is useless. Because you can add the result of ```hex.charAt(i)``` right away to the method ```indexOf(...)``` (below)